### PR TITLE
[FIX] mrp_brewing : missing lot number for sales orders

### PR DESCRIPTION
**Pull request by vvrossem**
Lot numbers added for sales orders lines

A lot number is provided only if : 
- there is a value for 'origin'
- there is a manufacturing order
- there is no manufacturing order but quant_ids

### CODE REVIEW

File: mrp_brewing/report/report_stock_finished_products.xml [hunk 1]
```diff
@@ -72,7 +72,18 @@
...
                                     <td>
-                                        <span t-esc="a.raw_material_production_id.brew_order_name"/>
+                                        <t t-if="a.origin">
+                                            <t t-set="lot_number" t-value="False"/>
+                                            <t t-if="a.raw_material_production_id.brew_order_name">
+                                                <t t-set="lot_number" t-value="a.raw_material_production_id.brew_order_name"/>
+                                            </t>
+                                            <t t-if="not a.raw_material_production_id.brew_order_name">
+                                                <t t-foreach="a.quant_ids" t-as="qids">
+                                                    <t t-set="lot_number" t-value="qids.lot_id.display_name"/>
```
Comment (houssine78) [inline mrp_brewing/report/report_stock_finished_products.xml]: it means that you're only taking the last one ?

Comment (houssine78) [reply mrp_brewing/report/report_stock_finished_products.xml]: from what I remember we told that if there is more than one lot number the lot number of each quant should be put in this td

**Review by houssine78 — COMMENTED** [review]
@vvrossem I have a question before approving this PR

File: mrp_brewing/report/report_stock_finished_products.xml (hunk 1 above)
Comment (vvrossem) [reply mrp_brewing/report/report_stock_finished_products.xml]: yes, it's the last lot number value that will appear on the report (I noticed it's always the same when multiple lot numbers). If there is a possibility to have different lot numbers, I change the behavior

**Comment by vvrossem** [comment]
@houssine78 
Lot numbers are now : 
- computed and stored depending on quant_ids
- formatted as : "lot_number_a/lot_number_b/.../lot_number_n"

File: mrp_brewing/models/stock.py [hunk 2]
```diff
@@ -27,6 +32,17 @@ def get_on_hand(self):
         string="Is Internal",
         compute="_compute_is_internal")
 
+    @api.multi
+    @api.depends('quant_ids')
+    def _compute_lot_numbers(self):
+
+        for stock_move in self:
+            lot_numbers = stock_move.lot_numbers.split('/') if stock_move.lot_numbers else []
```
Comment (houssine78) [inline mrp_brewing/models/stock.py]: no needs to initialize it before. just put a empty list

**Review by houssine78 — APPROVED** [review]
LGTM except for my small comments

File: mrp_brewing/models/stock.py (hunk 2 above)
Comment (vvrossem) [reply mrp_brewing/models/stock.py]: done

**Comment by houssine78** [comment]
👍 ok let's merge !

**Pull request merged**